Part3_icon-vis/ex3_0_icon-vis.ipynb

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
<p>This might not work well with the jupyterhub setup. As os.system might pickup system python rather than the one from the jupyter kernel. Can we include the code for this file in the notebook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Currently the exercise is set up in a way that the users can choose, which plot they want to try, i.e. the code of all the notebooks would have to be included if the exercise stays like this. Apart from that, the scripts are meant to be called from the terminal and not from Python, which is why I added this workaround. Furthermore, calling the scripts from Python would change the syntax, so the exercise wouldn't be useful to get familiar with icon-vis.</p><p> </p><p>Is there a way to use a bash terminal instead, where the environment is sourced? The users then would have to run the code from the bash terminal instead of from the notebook.</p>

Reply:
<p>What we can do is to run the script with the Python used in the environment. You can get the path using </p><pre class="ql-syntax" spellcheck="false">import sys
sys.executable
</pre><p> </p><p>In the current setup I get: '/.pyenv/versions/3.11.1/bin/python'</p><p> </p><p>You can then try to run the code using</p><pre class="ql-syntax" spellcheck="false">!/.pyenv/versions/3.11.1/bin/python script.py
</pre><p> </p><p>Please check and confirm if that works on our jupyterhub setup</p>

Reply:
It does work! I adapted the exercise accordingly.

Reply:
Super!